src/main/java/seedu/address/logic/parser/contactlistparsers/ContactListParser.java

```diff
@@ -67,9 +66,6 @@ public Command parseCommand(String userInput) throws ParseException {
         //case ExitCommand.COMMAND_WORD:
             //return new ExitCommand();
```

Comment:
If this is not needed, then we can just delete. I think there is a working ExitCommand outside in the main package already.

Reply:
Noted, will remove them in a subsequent PR.